README.md

```diff
@@ -58,16 +76,19 @@ Description for Network
 * `libvirt_auth`: connection url, by default: `qemu:///system`
 * `backup_dir`: directory for save backups, by default: `./`
 * `params`: params used for create object.
-  * `use_external_resource`: (optional) Use external object. The default is `false`.
-  * `resource_id`: (optional) Used to identify the object when `use_external_resource` is true.
+  * `use_external_resource`: (optional) Use external object. The default is
+    `false`.
+  * `resource_id`: (optional) Used to identify the object when
+    `use_external_resource` is true.
   * `dev`: Device name
   * `forwards`: settings for network `forwards`.
   * `ips`: settings for network `ips`.
 
 **Inputs for actions:**
 * `create`:
   * `params`: list of params for template, can be empty
-  * `network_file`: Template for network. Defaults is [network.xml](cloudify_libvirt/templates/network.xml)
+  * `network_file`: Template for network. Defaults is
+    [network.xml](cloudify_libvirt/templates/network.xml)
 
 **Runtime properties:**
 * `resource_id`: resource name.
```

Comment:
Seems like there are two `runtime_properties` sections in this readme.

Reply:
One section for domain, second one for network.